### ISSUE TITLE
option to resume playback at last played position

### ISSUE THREAD

**Issue by ahjolinna** [cmdrkotori/mpc-qt#111]
do I need to explain this?

**Comment by cmdrkotori** [comment]
Not at all, --no-resume-playback is a mpv option that I used to debug mpv configs. Making resuming possible would definitely come under expected behaviour for mpv users.

**Comment by ahjolinna** [comment]
so was that "yes" or a "no" :D

**Comment by cmdrkotori** [comment]
No need to explain, and yes it's possible.
